src/status_im/data_store/realm/contact_groups.cljs

```diff
@@ -42,7 +42,10 @@
 (defn- save-contacts
   [identities contacts]
   (doseq [contact-identity identities]
-    (.push contacts (clj->js {:identity contact-identity}))))
+    (if-let [contact (.find contacts (fn [object _ _]
```

Comment:
I thought for a some time how to do it :) but, 
```
(let [contact]
  (when-not (contact) 
```
It seemed to me not clear, i tried to find `when-not-let` and `if-let` with `nil` seemed fine  :)